app/apiProvider.js

```diff
@@ -8,6 +8,18 @@ export class ApiProvider {
     this.documentHost = documentHost;
   }
 
+  async getApiReadyHealth() {
```

Comment:
what's going on here. Why is this stuff in this PR?

Reply:
eugh must have branched off the wrong one

Reply:
closing this PR